docs/api/actions.md

```diff
@@ -104,6 +104,45 @@ function readPlanets () {
 repo.push(readPlanets)
 ```
 
+### Return a generator
+
+Often times we need to dispatch multiple actions in sequential
+order. For example, what if we want to ask the user to confirm their
+action before deleting a record?
+
+This can be accomplished by using a generator:
+
+```javascript
+function ask (message) {
+  return action => {
+    if (confirm(message)) {
+      action.resolve()
+    } else {
+      action.reject()
+    }
+  }
+}
+
+function deleteUser (id) {
+  return fetch.delete('/users/${id}').then(response => response.json())
+}
+
+function confirmAndDelete (user) {
+  return function * (send) {
```

Comment:
How's this feel?
https://github.com/vigetlabs/microcosm/blob/master/docs/api/actions.md#return-a-generator

https://github.com/vigetlabs/microcosm/blob/master/docs/guides/installation.md#extra-dependencies